charts/consul/test/unit/gateway-resources-job.bats

```diff
@@ -63,15 +63,14 @@ target=templates/gateway-resources-job.yaml
   [ "${actual}" = "true" ]
 }
 
-@test "apiGateway/GatewayClassConfig: custom configuration" {
+@test "apiGateway/GatewayClassConfig: custom configuration blah" {
```

Comment:
```suggestion
@test "apiGateway/GatewayClassConfig: custom configuration" {
```

Reply:
Yes oops, did this to be able to only run that test while testing.

Reply:
fixed